pkg/controller/ssp/metrics/reconcile.go

```diff
@@ -0,0 +1,71 @@
+package metrics
```

Comment:
I have a suggestion for a folder structure:
```
internal/pkg/operands/$operand/
                     | - reconcile.go
                     | - resources/
                          | - prometheusrules.go
                          | - deployments.go
                          | - services.go
                          | - $resourcename.go
```

and all resource generating functions like `newPrometheusRule` go in the respective resource file.
And we get the protection of not exporting those packages as a bonus.

What do you think?

Reply:
I would simplify it. If we have only one function for each resource type, we don't need to put them to separate files. They can all be in a single `resources.go` file.

And I'm not sure about using `internal` directory. Probably no one is going to import modules from `pkg/controller`.
I couldn't find any recommendations when to make packages internal. It would make sense if we were developing a library, but for an application, I don't know.

Reply:
Sure simplifying it to a single `resources.go` file is also a great alternative.
with regards to the `internal` folder, my approach is "unexport everything exept for...", I just think it makes more sense.

Reply:
Ok, I changed the directory structure.